website/content/labs/lab5.md

```diff
@@ -183,9 +183,10 @@ As above, you should use the TDD workflow when implementing these new functions.
 
 Finally, as before, you should include header comments in all the tests you write.
 
-# Submitting your lab
+## Submitting your lab
```

Comment:
In the lines right above this, do they need to do the same process of making a commit with just their tests before implementing the diameter, circumference, area, and overlap functions?

Reply:
That is what the "As above, you should use the TDD workflow when implementing these new functions." is driving at, but I've added some text to reiterate what they have to do.